docs/reference/contrib/modeladmin/create_edit_delete_views.rst

```diff
@@ -58,6 +58,38 @@ Or alternatively:
         # or
         edit_handler = TabbedInterface([ObjectList(custom_panels), ObjectList(...)])
 
+
+.. versionadded:: 2.5
+   ``edit_handler`` and ``panels`` can alternatively be 
+   defined on a ``ModelAdmin`` definition. This feature is especially useful
+   for use cases where you have to work with models that are 
+   'out of reach' (due to being part of a third-party package, for example).
+
+.. code-block:: python
+
+    class BookAdmin(ModelAdmin):
+        model = Book
+
+        panels = [
+            FieldPanel('title'),
+            FieldPanel('author'),
+        ]
+
+Or alternatively:
+
+
+.. code-block:: python
+
+    class BookAdmin(ModelAdmin):
+        model = Book
+
+        custom_panels = [
+            FieldPanel('title', classname='fn'),
```

Comment:
@mr-stateradio A small thing, but for maximum clarity, I think dropping the `classname` arguments for both these panels would be good too.